frontend/src/components/BigPopup/index.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { Drawer } from '@mui/material';
+import Box from '@mui/material/Box';
+
+interface PopupProps {
+  isOpen: boolean;
+  togleClose: any; // this function reatern set of useState
```

Comment:
try to use React.Dispatch<React.SetStateAction<boolean>> instead of any. If it does'nt work, remain as any